django/forms/widgets.py

```diff
@@ -594,7 +594,7 @@ def create_option(self, name, value, label, selected, index, subindex=None, attr
             option_attrs['id'] = self.id_for_label(option_attrs['id'], index)
         return {
             'name': name,
-            'value': value,
+            'value': force_text(value),
```

Comment:
There's [a ticket](https://code.djangoproject.com/ticket/27795) for auditing existing `force_text()` usage.